Add tip for handling "unable to login" issue. (#2414)
umbraco/UmbracoDocs: Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md

* Run the Umbraco 8 project locally
* The migration will need to be authorized - Cloud credentials is used for this

:::note

> If your login doesn't seem to be working, copy the attributes from your v7 web.config "UsersMembershipProvider" to the v8 web.config and try again:

Reviewer: ```suggestion
If your login doesn't seem to be working, try the following approach:

* Copy the `UsersMembershipProvider` attributes from your Umbraco 7 web.config, to the Umbraco 8 web.config
* Try again

Below is an example of how the attribute can look.
```

>        <add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="8" useLegacyEncoding="true" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" passwordFormat="Hashed" />

Reviewer: ```suggestion
```xml
<add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="8" useLegacyEncoding="true" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" passwordFormat="Hashed" />
```

```

> 

Reviewer: Making sure the note is closed.
```suggestion
:::
```
